UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AdminCommands;
+using TMPro;
+using UnityEngine;
+
+namespace AdminTools
+{
+	public class ServerSettingsPage : AdminPage
+	{
+		[SerializeField]
+		private TMP_InputField inputField = null;
+
+		public override void OnPageRefresh(AdminPageRefreshData adminPageData)
+    	{
+    		base.OnPageRefresh(adminPageData);
+            inputField.text = adminPageData.playerLimit.ToString();
+        }
+
+		public void OnChangeInputField()
+		{
+			if (int.TryParse(inputField.text, out var value))
+			{
+				AdminCommandsManager.Instance.CmdChangePlayerLimit(value);
```

Comment:
make sure it has a minimum value

Reply:
Technically doesn't really matter as negative numbers would be the same as 0. But i've added checks both side.